crates/types/src/config/mod.rs

```diff
@@ -35,22 +36,22 @@ pub use http::*;
 pub use ingress::*;
 pub use kafka::*;
 pub use metadata_store::*;
+pub use networking::*;
 pub use query_engine::*;
 pub use rocksdb::*;
 pub use worker::*;
 
 use std::path::PathBuf;
 use std::sync::Arc;
 
-use arc_swap::ArcSwap;
-use once_cell::sync::Lazy;
-use serde::{Deserialize, Serialize};
-use serde_with::serde_as;
-
 use super::live::{LiveLoad, Pinned};
 use crate::errors::GenericError;
 use crate::live::Live;
 use crate::nodes_config::Role;
+use arc_swap::ArcSwap;
+use once_cell::sync::Lazy;
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
```

Comment:
Why did you move the imports? We try to follow the following order:
```
std imports
<line break>
external crates imports
<line break>
restate crates imports
<line break>
local crate imports
```

Unfortunately this cannot be enforced by tools atm. However, once arranged in this form, `rustfmt` shouldn't change it.

Reply:
Wasn't intentional, I've now configured IDEA to leave these alone. I think it was the cause anyway; I'll pay more attention when I commit files - this slipped through.